Document/0x05d-Testing-Data-Storage.md

```diff
@@ -1045,6 +1045,9 @@ EditText.setEditableFactory(new Editable.Factory() {
 
 Refer to the `SecureSecretKey` example above for an example `Editable` implementation. Note that you will be able to securely handle all copies made by `editText.getText` if you provide your factory. You can also try to overwrite the internal `EditText` buffer by calling `editText.setText`, but there is no guarantee that the buffer will not have been copied already. If you choose to rely on the default input method and `EditText`, you will have no control over the keyboard or other components that are used. Therefore, you should use this approach for semi-confidential information only.
 
+In all cases, make sure that sensitive data in memory is cleared when a user signs out of the application. Last, make sure that highly sensitive information is cleared out the moment an Activity or Fragment hits `onPause`.
+Note: that this might mean that a user has to re-authenticate on every time he/she resumes the application.
```

Comment:
```suggestion
Note: that this might mean that a user has to re-authenticate every time he resumes the application.
```
I believe we're just using "he" throughout the guide.

Reply:
We are using mainly he in the guide for user. Let's stay with he to make it easier to read.

Reply:
No colon needed:

Note that this might mean that a user has to re-authenticate every time he resumes the application.